aviatrix/resource_aviatrix_firewall_instance.go

```diff
@@ -154,8 +154,11 @@ func resourceAviatrixFirewallInstanceCreate(d *schema.ResourceData, meta interfa
 	}
 
 	cloudType, err := client.GetVpcCloudTypeById(firewallInstance.VpcID)
+	if err == goaviatrix.ErrNotFound {
+		return fmt.Errorf("could not find the vpc with vpc_id=%s: %v", firewallInstance.VpcID, err)
+	}
```

Comment:
Move this ErrNotFound case inside err != nil

Reply:
Done 👍